components/gitpod-db/src/typeorm/team-db-impl.ts

```diff
@@ -377,10 +379,14 @@ export class TeamDBImpl implements TeamDB {
         }
     }
 
-    public async someOrgWithSSOExists(): Promise<boolean> {
-        const repo = await this.getOrgSettingsRepo();
+    public async hasAnyOrgWithActiveSSO(): Promise<boolean> {
+        const repo = await this.getTeamRepo();
         const result = await repo.query(
-            `select org.id from d_b_team as org inner join d_b_oidc_client_config as oidc on org.id = oidc.organizationId limit 1;`,
+            `select org.id from d_b_team as org inner join d_b_oidc_client_config as oidc on org.id = oidc.organizationId
```

Comment:
A bit unfortunate that server now talks to the OIDC table, which is owned by public api. We'll follow-up and clean-up this once we know better exactly how we can discover that state

Reply:
Yeah! The whole point to opt for server as home of this, is that it was initially called from `getLoggedInUser`. Now that this isn't the case, we can actually move it to papi. 🤔

Reply:
I'll look into rewriting in papi this one.

Reply:
@AlexTugarev Let's keep it like this. I think eventually this should be solvable with general APIs and proper Authorization which can grant the admin user access to all orgs to query.